src/api/mock-interceptor.ts

```diff
@@ -86,6 +86,7 @@ export const interceptRequestsOnMock = () => {
   mock
     .onGet(new RegExp(`${PublicationsApiPaths.PUBLICATION}/4327439`))
     .reply(200, { ...emptyRegistration, owner: 'tu@unit.no' });
+  mock.onGet(new RegExp(`${PublicationsApiPaths.PUBLICATION}/123`)).reply(200, mockPublishedRegistration);
```

Comment:
du vil ikke legge test-data-verdier i konstanter også ? ser dere bruker konstanter for data-testids

Reply:
jepp, skulle brukt id til `mockPublishedRegistration` her!